config/routes.rb

```diff
@@ -11,7 +11,7 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   resources :pages, only: %i[home] do
-    resources :food, only: %i[index show destroy] 
+    resources :food, only: %i[index new create destroy] 
```

Comment:
As stated in the requirements, the route to get the list of food is : 
<img width="295" alt="Capture d’écran 2022-05-19 à 00 19 30" src="https://user-images.githubusercontent.com/7243132/169164205-5afe7bb8-4595-4e73-b339-a6fda1bf0fde.png">

Please update the route to match the requirement